MVVMExample/MVVMExample/WorkersController.swift

```diff
@@ -28,13 +28,7 @@ class WorkersController: UIViewController {
             guard let strongSelf = self else { return }
             guard let text = tempText else { return }
             
-            strongSelf.viewModel.needfulWorkers = strongSelf.viewModel.workers.flatMap({ (worker) -> Worker? in
-                if worker.name.contains(text) || text == "" {
-                    return worker
-                }
-                return nil
-            })
-            
+            strongSelf.viewModel.fetchNeedfulWorkers(text: text)
             strongSelf.workersTable.reloadData()
```

Comment:
Please add handling of situation, when there is no changes in filtered workers, and there is no need in data reloading.